test/test_gpu3.cpp

```diff
@@ -8617,12 +8617,12 @@ TEST_F(NVFuserTest, AlignedSyncReduction1_CUDA) {
   fusion.addOutput(tv1);
 
   const int gdimx = 16;
-  const int bdimx = 100;
+  const int bdimy = 100;
```

Comment:
change to `bdimy` otherwise will generate warp reduction instead of block reduction.